### DISCUSSION TITLE
run failed on pull request

### DISCUSSION THREAD

**Discussion by rebullm** [greenelab/lab-website-template#190] [General] [answered]
Hi- 
UPDATE: seems to be working now, but weird that it showed an error and then worked anyway. Maybe I'm not waiting long enough? Thx!

I'm following the documentation and am on "configure your site" where I am making changes to _config.yaml. 
When looking at the page through deployed preview: https://rebullm.github.io/Bennett-Lab-website-v1.1/preview/pr-1/

it looks like my changes went through, so I merged pull requests, and received a run failed on pull request activity/ update-citations and then it skipped the build-preview. So the website doesn't reflect the current changes. 

Any ideas on what happened or how I can fix it? 
Thanks, 
Meg

**Comment by vincerubinetti** [comment, marked as answer]
It looks like the changes are [live to me. The new title and subtitle are there](https://rebullm.github.io/Bennett-Lab-website-v1.1/).

As for the [first failed action run](https://github.com/rebullm/Bennett-Lab-website-v1.1/actions/runs/4821706605), I believe I know why it happened, and [I've tracked it as a bug](https://github.com/greenelab/lab-website-template/issues/191). You can safely ignore it for now. But I'm talking only about this first failure. As for the other failures...

Make sure you are waiting for these GitHub Action runs (✅ ❌ ) to complete before merging your PRs. Do not merge with an error ❌ because it means there is likely a problem. For example, your recent PRs are failing because a few PRs back you made a change to [`_config.yaml` and made it invalid YAML](https://github.com/rebullm/Bennett-Lab-website-v1.1/blob/main/_config.yaml) (line 5).

Also, in `_config.yaml`, your `google-scholar` entry is incorrect. [Note the changes you made here](https://github.com/rebullm/Bennett-Lab-website-v1.1/pull/1/files). It needs to be just the ID, not the full URL. Be careful when replacing default values in the template. They are there as hints to show you the expected format of things.

**Reply by vincerubinetti** [reply]
Let me clarify. [Here are your runs](https://github.com/rebullm/Bennett-Lab-website-v1.1/actions). The ones that fail and say "pull request closed" are due to [a bug](https://github.com/greenelab/lab-website-template/issues/191) and can be ignored for now. All the other failures are due to changes you've made to the repo; something wrong with the repo that needs to be fixed. At the time of writing, what I'm seeing is that they're all due to the invalid config.yaml.
